### DISCUSSION TITLE
Selling Commands in QuickShop

### DISCUSSION THREAD

**Discussion by ZhiBuMiao** [QuickShop-Community/QuickShop-Hikari#1317] [Ideas]
I suggest incorporating a new feature within QuickShop, where infinite mode shops with a command block in the first slot could execute the command written in the command block's lore. Enabling this functionality would significantly enhance the plugin, allowing seamless execution of commands directly from shop displays, ultimately improving user experience and versatility within QuickShop.

**Comment by creatorfromhell** [comment]
> I suggest incorporating a new feature within QuickShop, where infinite mode shops with a command block in the first slot could execute the command written in the command block's lore. Enabling this functionality would significantly enhance the plugin, allowing seamless execution of commands directly from shop displays, ultimately improving user experience and versatility within QuickShop.

What would be the benefit? Buying commands?
